s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py

```diff
@@ -348,18 +348,21 @@ def get_probable_delete_index_id(self):
     def get_max_keys(self):
         """Return maximum number of keys from config file or KeyError."""
         max_keys = self.s3confstore.get_config('indexid>max_keys')
-        if max_keys:
+        if max_keys is not None:
```

Comment:
Max keys cannot be 0

Reply:
if there is no key indexid>max_keys then none is returned.